README-pt.md

```diff
@@ -38,13 +38,13 @@ Deseja levar seu aplicativo do Watson ao próximo nível? Quer aproveitar os ati
 ## 1. Clonar o repositório Clone o `watson-online-store` localmente.
 Em um terminal, execute `$ git clone https://github.com/ibm/watson-online-store`. Utilizaremos o arquivo [`data/workspace.json`](data/workspace.json) e a pasta [`data/ibm_store_html/`](data/ibm_store_html) 
 
-## 2. Criar serviços do Bluemix Crie os serviços a seguir: 
+## 2. Criar serviços do IBM Cloud Crie os serviços a seguir: 
```

Comment:
this would break the link from steps if it wasn't already broken by the translation